javascriptAttempt/recursion.test.js

```diff
@@ -0,0 +1,122 @@
+const recursion = require('./recursion');
```

Comment:
Nice work writing JavaScript tests!  I may, with your permission, swipe this as an option for future CS fun exercises.